_posts/2019-12-20-this-blog-post-contains-important-information-for-some-percentage-of-its-readers.markdown

```diff
@@ -0,0 +1,150 @@
+You asked the questions in class that surprised the teacher and annoyed your peers, and you probably got damn good
+grades.
+
+You may have been bullied.
+
+You likely went to college and focused on something that interested you -- perhaps you doubled down on language, or
+maybe you went into engineering or music or math. It was a revelation, because maybe for the first time in your
+life you were surrounded by a self-selecting group of peers, some of whom shared your passion for analysis and deep
+thinking. You didn't have to go to great lengths to establish a shared context, for instance. You could speak words
+that were meaningful to you and someone else could understand and offer a meaningful response, and this made you
+feel uncomfortable at first but you really loved it. Your experience of college was maybe a bit different than you
+expected - certainly nothing like the movies you watched as a kid. You got really into a weird hobby, maybe, and
+didn't end up going to too many parties.
+
+Relationships and intimacy have always been a bit fraught for you. You frequently find yourself trying to figure
+out what you're supposed to be feeling, and you work hard to show your partner the person you want to be. Maybe
+when you first moved in with someone you had a bit of a freakout that has made you nervous to think about ever
+since, or maybe meeting a significant other's family has been so anxiety-inducing as to prove impossible. Maybe you
+have a string of failed relationships behind you, each of them starting out so promisingly and then imploding
+catastrophically and confusingly once the initial honeymoon phase is over and it's time to really get to know and
+grow with another person. You don't know how to do that, and every time you try you seem to make someone upset.
+
+You're a software engineer at least partly because computers are much easier to communicate with than people. When
+a computer doesn't understand you it's because you've done something wrong. When a computer says something
+incorrect it's also because you've done something wrong. You can step through and rigorously debug the problem,
+identifying and isolating the mistake. The more you played around with tech the more you liked it - and when you
+discovered programming it blew your mind, because you _intuitively_ understood the relational nature of computation
+and found yourself suddenly gifted with superpowers.
+
+When you entered the working world you were REALLY confused, though. You were able to complete a week's worth of
+assignments in an afternoon. It was really obvious to you who was working intelligently and who was not, and you
+had to learn, perhaps painfully, that sharing that kind of feedback isn't always constructive. Maybe one day you
+got some criticism about something you did and it really hurt in ways you didn't expect -- maybe you found yourself
```

Comment:
I'm wary of having too many disjoint clauses joined by the same construction (`--` in this case). Maybe two sentences will flow better?

```suggestion
got some criticism about something you did and it really hurt in ways you didn't expect. Maybe you found yourself
```